packages/uicore/src/components/ConfirmDialog/ConfirmationDialog.tsx

```diff
@@ -90,6 +94,11 @@ export function ConfirmationDialog(props: ConfirmationDialogProps): React.ReactE
         className={css.body}>
         {contentText}
       </Layout.Vertical>
+      {children && (
+        <Container padding="none" className={childrenClassName}>
```

Comment:
why cann't user give same container with class name in the children, why we are making this tightly coupled

Reply:
Thanks. Addressed.